src/player.py

```diff
@@ -14,3 +14,6 @@ def update_statistics(self, statistics):
 
     def __lt__(self, other):
         return self.rating < other.rating
+
+    def __add__(self, other):
```

Comment:
I'm not sure if this is the correct code in terms of logic.
If we want to calculate the team's overall rating, it's better to do it in a separate team method.